docs/Installation.md

```diff
@@ -33,3 +33,48 @@ If `python` is not yet installed on your machine, you should add it.
 Error: connect EACCES /var/run/docker.sock
 ```
 Follow [these](https://docs.docker.com/install/linux/linux-postinstall/) instructions to create `docker` group and add your user.
+
+# Example Ubuntu Install
+This script automates steps 1-5 above and includes troubleshooting fixes. It was created for Ubuntu 18. 
```

Comment:
At the end of this paragraph I would add a sentence similar to: 

Copy and paste the following code into your text editor of choice, save it, and make it executable with `chmod +x myscript.bash`